opentracing/src/main/java/opentracing/propagation/SplitTextFormatInjector.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package opentracing.propagation;
+
+import java.util.Map;
+
+/**
+ *
+ */
+interface SplitTextFormatInjector extends Injector<Map> {
```

Comment:
It will probably save us some round-trips if I'm more specific about where I'm at with this...

We need to support the split carriers like we are doing in every other platform. (And the `Map<String, String>` carrier you have here is *not* a "split carrier" since the OpenTracing user cannot reference the two pieces independently.

BTW, I can see an argument to add required carrier format(s) for "non-split" binary and/or text data, but that would be *in addition to* (not instead of) the existing required split-text and split-binary formats.